torch/distributed/_fsdp/fully_sharded_data_parallel.py

```diff
@@ -95,16 +97,58 @@ class FullyShardedDataParallel(nn.Module):
             params and grads to be on same device to work with optimizer. This
             API is subject to change. Default is ``None`` in which case there
             will be no offloading.
+        auto_wrap_policy: (Optional [callable]):
+            A callable specifying a policy to recursively wrap layers with FSDP.
+            Note that this policy currently will only apply to child modules of
+            the passed in module. The overall module is always FSDP because we
```

Comment:
nit:  the passed in module. The reminder modules are always wrapped in the returned root FSDP instance